api-tests/api-audit/src/test/java/endpoints/dashboard/dashboardReview.feature

```diff
@@ -0,0 +1,15 @@
+Feature: dashboardReview
+
+  Background:
+    * header apiUser = 'apiadmin'
```

Comment:
See peerReview.feature comment on apiadmin and the additional statuses not checked here (ToDo)